src/main/kotlin/com/dansplugins/fiefs/fief/Fief.kt

```diff
@@ -0,0 +1,36 @@
+package com.dansplugins.fiefs.fief
+
+import java.util.UUID
+
+class Fief(private val name: String, private val ownerUUID: UUID) {
+    private var uuid = UUID.randomUUID()
```

Comment:
Can we make this model an immutable data class?
Additionally, perhaps we can document intent and add some type safety by using value classes for ids?

Reply:
Done and done!